.github/workflows/staging_deploy.yml

```diff
@@ -1,11 +1,6 @@
 name: Deploy to staging
 on:
   workflow_dispatch:
-  release:
-    branches:
-      - main
-    tags:
-      - v[0-9].[0-9]+.[0-9]+-staging
```

Comment:
Turns out the `release` trigger doesn't actually support filters, so there's no way to have this workflow run on any kind of `release` without also triggering the prod workflow. Since deploys to `staging` should mostly happen as part of the `prod` deploy workflow anyway, I've made this workflow manually triggered only.

Reply:
Interesting!